cloudformation/owasp-security-knowledge-framework.template.yaml

```diff
@@ -163,16 +181,20 @@ Resources:
       - Key: App
         Value:
           Fn::FindInMap: [ Constants, App, Value ]
+      - Key: Stack
+        Value:
+          Fn::FindInMap: [ Constants, Stack, Value ]
 
   LaunchConfig:
     Type: AWS::AutoScaling::LaunchConfiguration
     Properties:
       KeyName:
         Ref: KeyName
-      ImageId: ami-47a23a30
+      ImageId:
+        Ref: AMI
       SecurityGroups:
       - Ref: InstanceSecurityGroup
-      InstanceType: t2.nano
```

Comment:
I have found `t2.nano` instances don't have enough memory to build `lxml` from source. This caused sporadic failures during startup so I've bumped this to `t2.micro`.